### ISSUE TITLE
Tuya détecteur infra-rouge (id = TS0202, _TZ3040_bb6xaihh)

### ISSUE THREAD

**Issue by M4dd4M** [KiwiHC16/Abeille#2728]
Bonsoir !

Un autre détecteur infra-rouge de chez Tuya, dont le format est très sympa.
[discovery-TS0202__TZ3040_bb6xaihh.json](https://github.com/user-attachments/files/17298586/discovery-TS0202__TZ3040_bb6xaihh.json)

![discovery-TS0202__TZ3040_bb6xaihh](https://github.com/user-attachments/assets/397b5f79-84ad-453c-bcc7-749d5252c308)

Trouvé ici :  [https://fr.aliexpress.com/item/1005007024455870.html](url)

Si c'est possible de l'intégrer... un grand merci !

**Comment by tcharp38** [comment]
Ha ca doit etre ca le sujet sur lequel se focaliser en premier.

**Comment by M4dd4M** [comment]
@tcharp38 oui, c'est ça 😁, en espérant que je t'ai donné assez de billes du premier coup

**Comment by tcharp38** [comment]
Et celui la non plus n'a aucune reference ?
Sur le net cette signature semble etre celle d'un modele '809WZT'

**Comment by M4dd4M** [comment]
si, j'ai 809wz écrit en petit dans un coin (sur un autocollant du carton)

**Comment by tcharp38** [comment]
A decompresser dans Abeille/core/config/devices
[TS0202__TZ3040_bb6xaihh.zip](https://github.com/user-attachments/files/17340226/TS0202__TZ3040_bb6xaihh.zip)

et  l'image dans Abeille/images
![node_Tuya-PIR-MotionSensor](https://github.com/user-attachments/assets/6a0e804d-e466-47a5-8b14-a54793c0f55e)

Puis redemarrer Abeille
refaire inclusion
tester

Tout sera dans la prochaine beta

**Comment by M4dd4M** [comment]
Salut,
Désolé, retour tardif, mais j'ai un soucis avec ce capteur.
Il ne répond pas ... mais je pense que c'est parce que je suis au plafond en nombre d'objet possible.
Donc, standbye de mon coté tant que je n'aurai pas étendu ma ruche. 😐

**Comment by tcharp38** [comment]
Salut

Tu me files les logs ?
Qu'envisages tu pour étendre ton reseau du coup ?

**Comment by M4dd4M** [comment]
Salut, et bien y'a pas grand chose dans les logs justement, c'est pour ça que je met ça en standbye.
Je vais faire du ménage dans mes devices dans les semaines qui viennent, et recommencerai ça devrai être mieux.
J'ai pas encore choisi de système pour étendre .... même si j'avais commandé une pizigate+ avant de lite ton message :-( ... du coup je regarde du coté des zigate ethernet (pas encore trouvé) ... mais ça m'embêterai d'ajouter un autre système déporté de mon jeedom actuel.
(C'est frustant cette limite quand même.)

**Comment by tcharp38** [comment]
Toutes les clefs ont leur limite.
Et + c'est fonctionnel, plus tu as tendance à ajouter des capteurs :)

Si tu ne trouves pas de zigate eth tu peux la faire toi meme aussi. Si il suffit d un RPI (ou autre.. orange pi pour moi) et d'une clef (PI pour moi) et de n'installer que "socat". Donc si tu te sens de faire un peu de systeme.

**Comment by M4dd4M** [comment]
salut @tcharp38,
Bon, je continue le hors sujet, peut-être pourra tu m'aider ...
J'ai fait un peu de système ^^ :
 - Monté un RPi Zero avec une pizigate (v2) en GPIO
 - installé le tout en debian bookworm, patché pour désactiver le bluetooth
 - installé et configuré wiringpi, le GPIO, Socat
 - automatisé le démarrage de socat
 - démarré l'ajout d'une seconde ruche sur mon jeedom
 - la passerelle est bien reconnue, l'interrogation du port et du firmware sont OK depuis Jeedom (c'est là que je me suis rendu compte que c'est une pizigate v2 ...)
 Mais j'ai un soucis : 
- au déclenchement de l'inclusion sur la ruche 2, la pizigatev2 (du piZero) ne se met pas a clignoter
- et surtout cela bloque complètement la première ruche (qui ne répond plus à la découverte réseau) ... bref, plus rien n'est fonctionnel.
Je désactive la passerelle 2, et tout rentre dans l'ordre... 

Du coup j'ai plusieurs questions :
 - Aurais-tu une idée ou des pistes de recherche pour le soucis ?
 - Pourrais-tu détailler les soucis rencontrés avec les pizigate v2 ? car dans, en théorie, un simple remplacement de ma zigate v1 de ma ruche 1 par une v2 devrait lever toutes mes limitations de devices...

**Comment by tcharp38** [comment]
Salut
Histoire de ne pas pourrir ce sujet je te suggere d'en ouvrir un autre du style "Installation passerelle wifi + Piv2".

Donnes moi un package de logs que je vois ca de + pres.

**Comment by tcharp38** [comment]
Salut @M4dd4M 
J'imagine que ce sujet antique est "sans suite".
Je le ferme mais à réouvrir si il y a qqn interessé

Bonnes fetes
